src/app/visual/fonts/fonts.component.scss

```diff
@@ -0,0 +1,24 @@
+.icon::before {
```

Comment:
The point of these custom classes are to show that the pseudo-class technique still works for V4 and V5 icons.  It is mostly taken directly from the migration guide as something you'll have to fix from V4 -> V5.  https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4